Homework_6_Java_Intro/MyFirstJavaProject/src/Main.java

```diff
@@ -0,0 +1,15 @@
+//TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
+// click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
+public class Main {
+    public static void main(String[] args) {
+        //TIP Press <shortcut actionId="ShowIntentionActions"/> with your caret at the highlighted text
+        // to see how IntelliJ IDEA suggests fixing it.
+        System.out.printf("Hello pink Unicorn!");
```

Comment:
Hello!

![image](https://github.com/MartaM-a/-QA_Auto_Homework_1_Marta_M/assets/16562135/30f680ac-5615-48af-bdc8-5de38d067229)

Reply:
My girls like this one better
![image](https://github.com/MartaM-a/-QA_Auto_Homework_1_Marta_M/assets/127871105/596c1b2c-fd7b-46da-a5a2-dbca6e6161c6)

Reply:
Your unicorn is the cutest one, agreed :)